compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -127,14 +131,23 @@ class ReplDriver(settings: Array[String],
       else loop(interpret(res)(state))
     }
 
-    try withRedirectedOutput { loop(initState) }
+    try withRedirectedOutput { loop(initialState) }
     finally terminal.close()
   }
 
   final def run(input: String)(implicit state: State): State = withRedirectedOutput {
     val parsed = ParseResult(input)(state.context)
     interpret(parsed)
   }
+  
+  final def bindValues(bindPairs: ZippedTraversable2[String, Any]): State = {
```

Comment:
I'd rather expose `def bind(name: String, value: Any): State` as public API. This can be moved to `ConsoleInterface`.

I would make `initState` public final and rename it `initialState`